[2022-04-27 10:02:22,729] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: first.save scheduled__2022-04-25T00:00:00+00:00 [queued]>
[2022-04-27 10:02:22,740] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: first.save scheduled__2022-04-25T00:00:00+00:00 [queued]>
[2022-04-27 10:02:22,741] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-27 10:02:22,742] {taskinstance.py:1250} INFO - Starting attempt 1 of 2
[2022-04-27 10:02:22,742] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-27 10:02:22,759] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): save> on 2022-04-25 00:00:00+00:00
[2022-04-27 10:02:22,765] {standard_task_runner.py:52} INFO - Started process 3104 to run task
[2022-04-27 10:02:22,768] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first', 'save', 'scheduled__2022-04-25T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/first.py', '--cfg-path', '/tmp/tmppq3i04o3', '--error-file', '/tmp/tmp8rot_n4z']
[2022-04-27 10:02:22,769] {standard_task_runner.py:80} INFO - Job 8: Subtask save
[2022-04-27 10:02:22,827] {logging_mixin.py:109} INFO - Running <TaskInstance: first.save scheduled__2022-04-25T00:00:00+00:00 [running]> on host 65467f9f0591
[2022-04-27 10:02:22,884] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first
AIRFLOW_CTX_TASK_ID=save
AIRFLOW_CTX_EXECUTION_DATE=2022-04-25T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-25T00:00:00+00:00
[2022-04-27 10:02:22,885] {first.py:28} INFO - performing saving
[2022-04-27 10:02:22,886] {python.py:175} INFO - Done. Returned value was: None
[2022-04-27 10:02:22,897] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=first, task_id=save, execution_date=20220425T000000, start_date=20220427T100222, end_date=20220427T100222
[2022-04-27 10:02:22,941] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-27 10:02:22,991] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
